src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -365,46 +389,39 @@ private boolean checkSynchronized(final DetailAST synchronizedAst, boolean useBr
      * </pre>
      *
      * @param currentCase AST of the case that falls through to the next case.
-     * @param nextCase AST of the next case.
      * @return True if a relief comment was found
      */
-    private boolean hasFallThroughComment(DetailAST currentCase, DetailAST nextCase) {
-        boolean allThroughComment = false;
-        final int endLineNo = nextCase.getLineNo();
-
-        if (matchesComment(reliefPattern, endLineNo)) {
-            allThroughComment = true;
+    private boolean hasFallThroughComment(DetailAST currentCase) {
+        final DetailAST nextCase = currentCase.getNextSibling();
```

Comment:
Please rename this, such naming is misleading. We would not need the `if` statement below if we knew that `nextCase` naming is true, please just use `nextSibling`.

Reply:
nextCase -> nextSibling

Reply:
Done